includes/publisher/js/src/components/Text.jsx

```diff
@@ -0,0 +1,280 @@
+/* global atlasContentModelerFormEditingExperience */
+import React, { useState, useRef } from "react";
+import Icon from "../../../../components/icons";
+import { __ } from "@wordpress/i18n";
+const { wp } = window;
+import AddIcon from "../../../../components/icons/AddIcon";
+import TrashIcon from "../../../../components/icons/TrashIcon";
+
+export default function Text({ field, modelSlug, defaultError }) {
+	const [errors, setErrors] = useState({});
```

Comment:
This needs to receive the extra props.

This also lets us remove the `errors` useState, because errors is passed in from the parent component.

```suggestion
export default function Text({
	field,
	errors,
	validate,
	modelSlug,
	defaultError,
}) {
```